convert_to_tf_keras.py

```diff
@@ -0,0 +1,59 @@
+import os
+import sys
+
+list_conversions = [('import keras.', 'import tensorflow.keras.'),
+                    ('import keras ', 'from tensorflow import keras '),
+                    ('import keras\n', 'from tensorflow import keras\n'),
+                    ('from keras.', 'from tensorflow.keras.'),
+                    ('from keras ', 'from tensorflow.keras ')]
+
+
+def replace_imports(file_path, revert):

```

Comment:
Would be more prudent to add a test, to make sure the created file is as expected (like we do with generated docs)

Reply:
We already run all the tests with the generated files. Do you have a specific test in mind?

Reply:
Ho, I see what you mean. I'm not in favor of including tests at running time (except for catching user errors) because this will complexify the codebase. Separating the tests and the implementation allow for a better readability. And when tests fail, we know exactly which use case caused it. It's more extensible.

Reply:
@RaphaelMeudec do you have any simple tests in mind?

Reply:
@gabrieldemarmiesse What I had in mind is :

- Create an asset file in the test directory, and apply `replace_imports` to it. Make sure that there is an output file created, and that the return value of the function is approriate
- Apply in reverse the same function to make sure generated file matches the original file

Reply:
I added a test in the same file (we don't have a choice, we can't import the function directly). And added the command to run it in travis.